docs/v3/source/includes/resources/deployments/_header.md

```diff
@@ -13,6 +13,7 @@ Deployments are different than the traditional method of pushing app updates whi
 Deployment strategies supported:
 
 * [Rolling deployments](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html) allows for
-applications to be deployed without incurring downtime by gradually rolling out instances.
+applications to be deployed without incurring downtime by gradually rolling out instances. Max-in-flight can be configured
```

Comment:
sounde like we want to update the linked rolling-deploy doc with max-in-flight examples, but that's a separate story

Reply:
Makes sense. Will have a follow up issue for this. Resolving for this PR.